Documentation/ApiOverview/Events/Events/Core/Imaging/ModifyRecordOverlayIconIdentifierEvent.rst

```diff
@@ -0,0 +1,36 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; ModifyRecordOverlayIconIdentifierEvent
+..  _ModifyRecordOverlayIconIdentifierEvent:
+
+======================================
+ModifyRecordOverlayIconIdentifierEvent
+======================================
+..  versionadded:: 13.0
+
+    The PSR-14 event :php:`ModifyRecordOverlayIconIdentifierEvent` has
+    been introduced which serves as a more flexible replacement for the
+    removed hook:
+
+    *   :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['TYPO3\CMS\Core\Imaging\IconFactory']['overrideIconOverlay']`
```

Comment:
The list is not necessary as it is only one item, so I suggest to write:

    removed hook :php:`...`

without the list.